scapy/contrib/cansocket_python_can.py

```diff
@@ -115,8 +73,8 @@ def __init__(self):
         self.pool = dict()  # type: Dict[str, SocketMapper]
         self.pool_mutex = threading.Lock()
 
-    def internal_send(self, sender, msg, prio=0):
-        # type: (SocketWrapper, can_Message, int) -> None
+    def internal_send(self, sender, msg):
```

Comment:
_internal_send() might be a better name.

Reply:
Thanks